packages/dvilela/connections/tweepy/connection.py

```diff
@@ -337,4 +337,11 @@ def unfollow_by_id(self, user_id: str) -> Dict:
 
     def get_me(self) -> Optional[Dict]:
         """Get own user information"""
-        return self.twitter.get_me()  # type: ignore
+        if self.twitter is None:
```

Comment:
Another unauthorized, undiscussed change in a commit that is titled "fix: linters"? That's not acceptable anymore! 
Also, super bad design. Why add this here? We alre already taking care of this scenario for ALL methods in line 237. So, under which circumstances could self.twitter be None here?